src/renderer/index.css

```
@@ -257,16 +281,53 @@ textarea::placeholder {
+.bounce-dots {
+	display: inline-flex;
+	align-items: center;
+}
+
+.bounce-dots span {
+	display: inline-block;
+	width: 6px;
+	height: 6px;
+	border-radius: 9999px;
+	margin: 0 2px;
+	background-color: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)